### PR TITLE
Returned aria-posinset and aria-setsize to commandbar

### PR DESCRIPTION
**Pull request by micahgodbolt**

### CODE REVIEW

File: packages/office-ui-fabric-react/src/components/CommandBar/CommandBar.base.tsx [hunk 1]
```diff
@@ -77,6 +78,8 @@ export class CommandBarBase extends BaseComponent<ICommandBarProps, {}> implemen
       cacheKey: '',
     };
 
+    this._setAriaPosinset(commandBardata);
```
Comment (dzearing) [inline packages/office-ui-fabric-react/src/components/CommandBar/CommandBar.base.tsx]: nit... `commandBardata`? Bardata is not a word :)

**Comment by betrue-final-final** [comment]
It looks like you may need to pull from master as there are regressions to split button in here.

**Comment by micahgodbolt** [comment]
This is the 6.0 branch. Odd that there are split button regressions. Especially for the small change I made to just commandBar

**Comment by micahgodbolt** [comment]
I'm up to date with 6.0 branch too. So a) it's a random flicker or b) screener is comparing against the wrong branch. Regardless this is just 6.0. If master is different, we'll reconcile those changes when 6.0 is released.

**Comment by micahgodbolt** [comment]
There is no code in this PR that is affecting the button styles. Either the baselines were incorrect, or there is some other bug. But seeing as this is a push to 6.0, and that the missing CSS class is breaking other's 6.0 builds, I'd like to get this pushed in and we'll resolve the button issue when we finally get this into master (if its still an issue). Okay with that @betrue-final-final and @dzearing ?

**Comment by betrue-final-final** [comment]
That makes me nervous, if you can understand that, and if I okay that change then screener will think that the wrong visual is correct.

**Comment by micahgodbolt** [comment]
Point is that I can't sync with 6.0 (nothing to pull), and no code I changed would have this effect on the button. I can only assume that the 6.0 branch's baselines aren't updated.

**Comment by UMAknow-cslecours** [comment]
See this commit https://github.com/OfficeDev/office-ui-fabric-react/commit/7739825ca44396f4b6a74892e1b57552ea1e3c7a for the current failing screener tests.

**Comment by cslecours** [comment]
See this commit https://github.com/OfficeDev/office-ui-fabric-react/commit/7739825ca44396f4b6a74892e1b57552ea1e3c7a for the current failing screener tests. (It was fixed around an hour ago by @dzearing.

**Comment by micahgodbolt** [comment]
I re-ran the 6.0 branch locally, and the two buttons are showing up this way in 6.0 as well. I'm not sure why the baselines aren't properly updated, but there is nothing about this PR that is making those buttons fail. We need to merge this in to unblock other 6.0 work and then figure out why those baselines are coming back that way.

image

**Comment by micahgodbolt** [comment]
i take that last bit back..i need to rebuild

**Comment by micahgodbolt** [comment]
Ok, previously deleted comment stands. This is how the buttons look in 6.0. For some reason the baselines are out of sync

image

We need to merge this in (as it's blocking other PRs) and figure out why these buttons are testing this way. I can confirm that they work properly when ran locally in storybook.

**Comment by betrue-final-final** [comment]
You're saying that the actual code doesn't look like what we have in screener?

**Comment by micahgodbolt** [comment]
Accepting this PR won't change what our baselines are, it'll simply merge the code in and new baselines will be created.

**Comment by betrue-final-final** [comment]
Do you mean accepting the screener tests or ramming through the PR over them?

**Comment by micahgodbolt** [comment]
I'm saying that my code has zero effect on the button component, and that accepting the screener tests doesn't actually change anything in screener. It only lets the test pass so that code can be merged. An update to the screenshots is run when the code is merged into the branch (so if tests are accepted but PR takes a week to merge, baselines aren't out of sync).

I'm going to merge the code in and we can investigate why the tests are flakey in 6.0.


Sent from Outlook<http://aka.ms/weboutlook>

________________________________

**Pull request merged**